svelte-kit-scss/src/routes/(authenticated)/[username]/[repo]/+layout.server.ts

```diff
@@ -1,29 +1,50 @@
 import { ENV } from '$lib/constants/env';
 import type { LayoutServerLoad } from './$types';
-import type { ReadmeApiResponse, RepoApiResponse, RepoContentsApiResponse } from '$lib/interfaces';
-import { mapRepoResToRepoState } from '$lib/helpers';
+import type {
+  PullRequestAPIResponse,
+  // ReadmeApiResponse,
+  RepoApiResponse,
+  // RepoContentsApiResponse
+} from '$lib/interfaces';
+import { mapRepoResToRepoState, remapRepoPullRequestCollection } from '$lib/helpers';
 
 export const load: LayoutServerLoad = async ({ params, fetch }) => {
   const { username, repo } = params;
   const getRepoUrl = new URL(`/repos/${username}/${repo}`, ENV.GITHUB_URL);
-  const getRepoPullsUrl = new URL(`/repos/${username}/${repo}/pulls`, ENV.GITHUB_URL);
-  const getRepoContentsUrl = new URL(`/repos/${username}/${repo}/contents`, ENV.GITHUB_URL);
-  const getRepoReadmeUrl = new URL(`/repos/${username}/${repo}/readme`, ENV.GITHUB_URL);
 
-  const [repoData, repoPRs, repoContentsData, repoReadmeData] = await Promise.all([
+  const getOpenRepoPullsUrl = new URL(`/search/issues`, ENV.GITHUB_URL);
+  getOpenRepoPullsUrl.searchParams.append('q', `repo:${username}/${repo} is:pr is:open`);
+  getOpenRepoPullsUrl.searchParams.append('per_page', '1');
+
+  // const getRepoContentsUrl = new URL(`/repos/${username}/${repo}/contents`, ENV.GITHUB_URL);
+  // const getRepoReadmeUrl = new URL(`/repos/${username}/${repo}/readme`, ENV.GITHUB_URL);
```

Comment:
Is this a WIP, do these comments need to be removed, or are you leaving them in because they will be uncommented in a later PR?